### PR TITLE
Update notifications.json

### PR DESCRIPTION
**Pull request by ViktarStarastsenka**

### CODE REVIEW

File: notifications.json [hunk 1]
```diff
@@ -1,3 +1,11 @@
 {
-  "notifications": []
+  "notifications": [
+          {
+            "title": "Have an idea or a bug?",
```
Comment (K-Jo) [inline notifications.json:4]: feature not idea

File: notifications.json [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+            "body": "Would you like to see specific features added or have a bug to report? <p> <a href="https://github.com/RedisInsight/RedisInsight">Share</a> it with us! <p> And star a repository to support the project!",
```
Comment (K-Jo) [inline notifications.json:5]: And star the repository if you like RedisInsight

**Review by K-Jo — CHANGES_REQUESTED** [review]

File: notifications.json [hunk 3]
```diff
@@ -1,3 +1,11 @@
 {
-  "notifications": []
+  "notifications": [
+          {
+            "title": "Have a feature or a bug?",
+            "body": "Would you like to see specific features added or have a bug to report? <p> <a href="https://github.com/RedisInsight/RedisInsight">Share</a> it with us! <p> And star the repository if you like RedisInsight!",
```
Comment (arthosofteq) [inline notifications.json:5]: This is not a valid json. We have tests failed. Please escape `"` char with `\"` inside strings.
`"Would you like to see specific features added or have a bug to report? <p> <a href="https://github.com/RedisInsight/RedisInsight">Share</a> it with us! <p> And star the repository if you like RedisInsight!",`
should be 
`"Would you like to see specific features added or have a bug to report? <p> <a href=\"https://github.com/RedisInsight/RedisInsight">Share</a> it with us! <p> And star the repository if you like RedisInsight!\"",`

**Review by arthosofteq — CHANGES_REQUESTED** [review]
I highly recommend to use editor that supports json or some online json validator like https://jsonlint.com/

File: notifications.json (hunk 3 above)
Comment (arthosofteq) [reply notifications.json:5]: UPD: Here is the correct json with `"` escaped
`"Would you like to see specific features added or have a bug to report? <p> <a href=\"https://github.com/RedisInsight/RedisInsight\">Share</a> it with us! <p> And star the repository if you like RedisInsight!"`

**Review by arthosofteq — APPROVED** [review]

File: notifications.json [hunk 4]
```diff
@@ -1,3 +1,11 @@
 {
-  "notifications": []
+  "notifications": [
+          {
+            "title": "Have a feature or a bug?",
```
Comment (K-Jo) [inline notifications.json:4]: Missing feature or found a bug?

**Review by K-Jo — APPROVED** [review]

**Review by arthosofteq — APPROVED** [review]

**Review by arthosofteq — APPROVED** [review]

**Pull request merged**